src/test/scala/org/ergoplatform/utils/ErgoTestConstants.scala

```diff
@@ -41,4 +43,7 @@ trait ErgoTestConstants extends ScorexLogging {
   val defaultExtension: ExtensionCandidate = ExtensionCandidate(Seq(), Seq((EmptyDigest32, EmptyDigest32)))
   val emptyExtension: ExtensionCandidate = ExtensionCandidate(Seq(), Seq())
 
+  val defaultTimeout: Timeout = Timeout(14.seconds)
```

Comment:
 This probably should be `def` to fix initialization order for the line below. Maybe `lazy val` will also suit